packages/flutter_adaptive_scaffold/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 * Updates minimum supported SDK version to Flutter 3.22/Dart 3.4.
 * Updates minimum supported SDK version to Flutter 3.19/Dart 3.3.
+* Expose labelType for NavigationRails.
```

Comment:
super nit: labelType, navigationRailDestinationBuilder, groupAlignment, and navRailTheme should be surrounded by backticks, similar to other parameters in this file. `parameter`.